source/partials/next_release/_release.md.erb

```diff
@@ -1,51 +1,21 @@
-<h4>Security Fixes</h4>
+<h4>Bundled Plugins</h4>
 
-This release has critical security fixes to prevent CSRF, XSS and Session hijacking exploits. We highly recommend users to upgrade to keep your GoCD server and agent secure.
-Thanks [4cad](https://hackerone.com/4cad) for reporting these issues. 
+The [JSON config repo plugin](https://github.com/tomzo/gocd-json-config-plugin) and the [YAML config repo plugin](https://github.com/tomzo/gocd-yaml-config-plugin) are bundled with the GoCD server. With this release, support for configuring config repository is added by default.
+These plugins are maintained by our core contributor [Tomasz Sętkowski](https://github.com/tomzo).
+Please raise any issues found in the relevant repositories.
 
 <h4>Improvements</h4>
 
-* <%= link_to_commit '42c681d1585db99747877830c2b1fdc2ca3e80b1', 'Added option to skip agent startup on windows.', 3577 %>
+* <%= link_to_commit '1ccb0294265b4baea091c9068d7813fdde5009b4', 'Ensure order of auth_config is used to authenticate users', 3608 %>
```

Comment:
Ensure order of auth_config in config xml is used to derive the order in which plugins are used for authentication.